en/docs/administer/logging-and-monitoring/monitoring/monitoring-server-health.md

```diff
@@ -1,118 +1,163 @@
 # Monitoring Server Health
 
-!!! warning
-This document is work in progress!
+**Health Check API** is used to validate and verify the status of the APIM server and its dependencies
+It is a REST service which returns the operational status, performance information of the configured endpoints with appropriate HTTP return code and a JSON object
```

Comment:
What does ths JSON Object include?

Reply:
```
{
       "health":[
          {
             "key":"jdbc/SHARED_DB.connectivity.time.ms",
             "value":"0"
          },
          {
             "key":"WSO2MBStoreDB.connectivity.time.ms",
             "value":"0"
          },
          {
             "key":"jdbc/WSO2CarbonDB.connectivity.time.ms",
             "value":"0"
          },
          {
             "key":"jdbc/WSO2AM_DB.connectivity.time.ms",
             "value":"0"
          },
          {
             "key":"jdbc/WSO2CarbonDB.active.connection.count",
             "value":"1"
          },
          {
             "key":"jdbc/WSO2MetricsDB.active.connection.count",
             "value":"1"
          },
          {
             "key":"jdbc/WSO2MetricsDB.connectivity.time.ms",
             "value":"33"
          },
          {
             "key":"jdbc/WSO2AM_DB.active.connection.count",
             "value":"1"
          },
          {
             "key":"jdbc/SHARED_DB.active.connection.count",
             "value":"1"
          },
          {
             "key":"WSO2MBStoreDB.active.connection.count",
             "value":"1"
          }
       ]
    }
```

It has been included in the latter part of this doc.